packages/cli/src/cmds/preview/constants.ts

```diff
@@ -64,7 +64,7 @@ export const funcHostedBotStartPattern =
   /Worker process started and initialized|Host lock lease acquired by instance ID/g;
 export const funcHostedBotAzuritePattern = /successfully listening/g;
 export const gulpServePattern = /^.*Finished subtask 'reload'.*/g;
-export const defaultRunningPattern = /started|successfully|finished|crashed|failed/i;
+export const defaultRunningPattern = /started|successfully|finished|crashed|failed|listening/i;
```

Comment:
LGTM. @swatDong Any concerns?

Reply:
The CLI source code looks good to me.
BTW, @hund030 - any template or sample's `tasks.json` needs update?

Reply:
We added `listening` in tasks.json on the initial day. But the CLI preview was unintentionally omitted.